packages/language-server/test/lib/documents/utils.test.ts

```diff
@@ -129,22 +129,50 @@ describe('document/utils', () => {
                     </script>
                 {/await}
                 <p>{@html <script> consolelog('not top level')</script>}</p>
+                {@html mycontent}
+                {@debug myvar}
                 <!-- p{ color: blue; }</script> -->
                 <!--<script lang="scss">
                 p{ color: blue; }
                 </script> -->
-                <scrit>blah</script>
+                <scrit>blah</scrit>
                 <script>top level script</script>
             `;
+            // Note: cannot test <scrit>blah</scriPt> as that breaks parse5 parsing for top level script!
```

Comment:
this used to work under the old regex impl, but doesn't work when using parse5, because the `</script>` causes parse5 to swallow the subsequent `<script>top level script</script>`, making it undetectable for our purposes. Better to abandon support for this funky error case imo and have a better overall parsing strategy.